README.md

```diff
@@ -1,2 +1,40 @@
-# Restaurant-Page
-This project is about DOM manipulation by dynamically rendering a simple restaurant homepage using Javascript alone to generate its entire content.
+# Restaurant Page
+A simple restaurant site built using Javascript alone to generate its entire content.
+
+![screenshot 1](src/images/home.png)
+
+## Built With
+
+- JavaScript
+- Webpack
+- HTML/CSS
+
+[Live Demo Link](https://raw.githack.com/Mupa1/Restaurant-Page/restaurant_page/dist/index.html)
+
+## Getting Started
+To get a copy up and running follow these simple example steps:-
+- Clone `https://github.com/Mupa1/Restaurant-Page.git` to your local machine.
+- cd into the Restaurant-Page directory and navigate into the dist directory.
+- Run index.html in a browser.
```

Comment:
For development purposes, add instruction to run _npm install_ and _npm run build_

Reply:
Done! Thank you for your feedback!